kiwmi/luak/kiwmi_keyboard.c

```diff
@@ -17,22 +17,44 @@
 #include "luak/kiwmi_lua_callback.h"
 
 static int
-l_kiwmi_keyboard_configure(lua_State *L)
+l_kiwmi_keyboard_set_keymap(lua_State *L)
```

Comment:
Maybe just `keymap` as the current API doesn't use any set, or get stuff. Not unlikely that that will change in the future, but let's keep things consistent for now.